docs/extenders/SecureCredentials.md

```diff
@@ -0,0 +1,27 @@
+# Secure Credentials API Adoption
+
+Extenders may have a need for z/OS actions to be taken outside of Zowe Explorer using Zowe profiles that are managed by the Secure Credentials Store. Zowe Explorer API has a way for extenders to handle this situation.
```

Comment:
We are starting this article with what's essentially a note. Maybe we can rearrange this somehow? 

I would start the article with a description of how to adopt the Secure Credentials API. For example,

As a Zowe Explorer developer/user, use the secure credentials API to {achive a certain goal/goals}. {Short sentence about the business value. In other words, the reason customers want/need to use this API to begin with}.